cgi-bin/DW/FeedCanonicalizer.pm

```diff
@@ -318,26 +318,16 @@ sub make_ljish {
     $username = lc($username);
     $username =~ s/-/_/g;
 
-    my $extra = "";
-    if ( $feed =~ m/customview$/i ) {
-        $extra = $extra_raw if $extra_raw =~ m!^/!;
-        return LJ::create_url("/$username/customview$extra",
-            proto => "ljish",
-            host => $domain,
-            cur_args => { $uri->query_form },
-            keep_args => [ qw/ styleid show filter / ] );
-    } else {
-        my $extra;
-        $extra = "/friends" if $feed =~ /friends$/;
-
-        my %query = $uri->query_form;
-
-        if ( $query{tag} ) {
-            return "ljish://$domain/$username$extra?tag=" . uri_escape( $query{tag} );
-        }
+    my $extra;
```

Comment:
This should keep the ="" assignment to avoid warnings about using undefined variables in string concatenation.